### DISCUSSION TITLE
Suspected Typos

### DISCUSSION THREAD

**Discussion by TullyMonster** [biostars/biostar-handbook-issues#302] [General]
[First impression`f` of datasets](https://www.biostarhandbook.com/ncbi-datasets.html#:~:text=First%20impressionf%20of%20datasets)

**Comment by ialbert** [comment]
to report typos, please collect these and send out several such typose at a time to the contact@biostarhandbook.com email

the issue tracker is for substantive discussions that relate to the scientific content and/or need clarifications

**Reply by TullyMonster** [reply]
I got it.
